src/amqp_sender.cpp

```diff
@@ -0,0 +1,94 @@
+#include "irods/private/audit_amqp.hpp"
+#include "irods/private/amqp_sender.hpp"
+
+#include <boost/config.hpp>
+
+namespace irods::audit
+{
+	namespace {
+		BOOST_FORCEINLINE void log_proton_error(const proton::error_condition& err_cond, const std::string& log_message)
+		{
+			// clang-format off
+			log_re::error({
+				{"rule_engine_plugin", rule_engine_name},
+				{"log_message", log_message},
+				{"error_condition::name", err_cond.name()},
+				{"error_condition::desc", err_cond.description()},
```

Comment:
i see this is just 'moved code'... but is this a standard thing?   desc?    rather than description?

Reply:
no, was introduced in the additional logging for proton commit. I abbreviated it for alignment with `name`, but I don't mind changing it.

Reply:
i vote more letters unless there is a standing reason/standard/community that does it an existing way.  we live with these strings forever...